Cartfile

```diff
@@ -1,7 +1,8 @@
-binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 5.0.0
+binary "https://mapbox.s3.amazonaws.com/mapbox-accounts/ios/MapboxAccounts.json" ~> 0.0.1
+binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 5.1
 binary "https://www.mapbox.com/ios-sdk/MapboxNavigationNative.json" ~> 6.2.1
-github "mapbox/MapboxDirections.swift" ~> 0.28.0
+github "mapbox/MapboxDirections.swift" ~> 0.29
 github "mapbox/turf-swift" ~> 0.3
-github "mapbox/mapbox-events-ios" ~> 0.9.3
+github "mapbox/mapbox-events-ios" "fred/allow-subclassing"
```

Comment:
Don't forget to pin to a proper release when https://github.com/mapbox/mapbox-events-ios/pull/139 has been solved.